lib/truncate.js

```diff
@@ -8,123 +9,141 @@ exports.span = truncSpan
 exports.error = truncError
 
 function truncMetadata (metadata, opts) {
-  metadata.process.title = truncate(String(metadata.process.title), opts.truncateKeywordsAt)
+  breadthFilter(metadata, (value, key, path) => {
+    if (typeof value !== 'string') {
+      return value
+    }
+
+    let max = opts.truncateStringsAt
```

Comment:
Could be implemented with a ternary operator instead. But maybe V8 is able to optimize it to that automatically

Reply:
fixed